corehq/apps/app_manager/tests/data/suite/search_command_detail.xml

```diff
@@ -5,7 +5,7 @@
         <locale id="m0.case_short.title"/>
       </text>
     </title>
-    <field sort="default">
+    <field>
```

Comment:
huh, that's strange. I think that's fine thanks. (i added that because otherwise `wrap` [tries to hit couch](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L4669) which makes it fail as a `SimpleTestCase`... not sure if there was an easier way to address that)